gis4wrf/core/constants.py

```diff
@@ -17,6 +17,7 @@ class ProjectionTypes(object):
     MERCATOR = 3
     LAT_LON = 6
 
+WRF_WPS_DIST_OLD_VERSIONS = ['3.9']
 WRF_WPS_DIST_VERSION = '4.0'
```

Comment:
I think that's fine, we can follow the [semantic versioning](https://semver.org/) syntax for that, in particular the build metadata part. It's not really meant for that, but it's better than nothing. So, it could be `4.0+1`, `4.0+2` etc. This would then be displayed everywhere (download button, download confirmation, update reminder,..). Do you think that's ok?

Reply:
I think that's fine considering that the pre-built binary releases should correspond to source release.
So as it just temporary, I'm OK with it.